light-node/src/config.rs

```diff
@@ -89,6 +92,7 @@ impl Default for LightNodeConfig {
             // TODO(ismail): need at least 2 peers for a proper init
             // otherwise the light node will complain on `start` with `no witness left`
             light_clients: vec![LightClientConfig::default()],
+            shared_state_config: "./lightstore".parse().unwrap(),
```

Comment:
Not sure I have the right context but doesn't this mean you can remove the separate stores in each light client?
See above:
```rust
/// LightClientConfig contains all options of a light client instance.
#[derive(Clone, Debug, Deserialize, Serialize)]
#[serde(deny_unknown_fields)]
pub struct LightClientConfig {
    /// Address of the Tendermint fullnode to connect to and
    /// fetch LightBlock data from.
    pub address: tendermint::net::Address,
    /// PeerID of the same Tendermint fullnode.
    pub peer_id: PeerId,
    /// The data base folder for this instance's store.
    pub db_path: PathBuf,
}
```

Reply:
This is something I wanted to discuss. In practice we can use in-memory state for each peer instance enabling us to remove the separate stores for each light client. I'm however unsure if we might still want to keep these stores to be able to inspect what each light client verified by itself(do we even want such a feature?). @liamsi

Reply:
>  I'm however unsure if we might still want to keep these stores to be able to inspect what each light client verified by itself(do we even want such a feature?).

Not sure tbh, but there might be some cases where this is required for the fork-detector!